### PR TITLE
Nepali number count spelling corrected

### PR DESCRIPTION
**Pull request by codernirdesh**

### CODE REVIEW

File: lib/src/constants.dart [hunk 1]
```diff
@@ -16,46 +16,46 @@ class Constants {
...
+    "बाईस",
     "तेइस",
     "चौबीस",
-    "पचीस",
-    "छब्बीस",
-    "सत्ताइस",
-    "अठ्ठाइस",
+    "पच्चिस",
+    "छब्बिस",
+    "सत्ताईस",
```
Comment (subarnaPdl) [inline lib/src/constants.dart:36]: i think it should be 'सत्ताइस' instead of 'सत्ताईस'

File: lib/src/constants.dart [hunk 2]
```diff
@@ -16,46 +16,46 @@ class Constants {
...
     "चौध",
     "पन्ध्र",
     "सोह्र",
     "सत्र",
-    "अठाह्र",
+    "अठार",
     "उन्नाइस",
     "बीस",
-    "एकाइस",
-    "बाइस",
```
Comment (subarnaPdl) [inline lib/src/constants.dart:31]: 'बाइस' is already correct here

File: lib/src/constants.dart [hunk 3]
```diff
@@ -16,46 +16,46 @@ class Constants {
...
-    "छयालिस",
-    "सरचालीस",
-    "अरचालीस",
-    "उननचास",
+    "बयालीस",
+    "त्रिचालीस",
+    "चौवालीस",
+    "पैँतालीस",
+    "छयालीस",
+    "सड्चालीस",
```
Comment (subarnaPdl) [inline lib/src/constants.dart:56]: i think it should be 'सच्चालीस' instead of 'सड्चालीस'

File: lib/src/constants.dart [hunk 4]
```diff
@@ -16,46 +16,46 @@ class Constants {
...
+    "अठचालीस",
+    "उनन्पचास",
```
Comment (subarnaPdl) [inline lib/src/constants.dart:58]: i think it should be 'उनन्चास' instead of 'उनन्पचास'

**Review by subarnaPdl — COMMENTED** [review]
We can write both 'अरब' and 'अर्ब' and similarly both 'खरब' and 'खर्ब'.

**Comment by codernirdesh** [comment]
The common practice is "अर्ब" & "खर्ब" in all the places.

**Comment by subarnaPdl** [comment]
Ok, Thanks for the PR! I'll check and merge it soon.

File: lib/src/constants.dart [hunk 5]
```diff
@@ -16,46 +16,46 @@ class Constants {
     "सात",
     "आठ",
     "नौ",
-    "दस",
```
Comment (subarnaPdl) [inline lib/src/constants.dart:19]: Both of them either 'दस' or 'दश', are correct here.

File: lib/src/constants.dart (hunk 2 above)
Comment (subarnaPdl) [inline lib/src/constants.dart:31]: 'बाइस' is already correct here.

File: lib/src/constants.dart [hunk 6]
```diff
@@ -16,46 +16,46 @@ class Constants {
...
     "पन्ध्र",
     "सोह्र",
     "सत्र",
-    "अठाह्र",
+    "अठार",
     "उन्नाइस",
     "बीस",
-    "एकाइस",
-    "बाइस",
+    "एक्काईस",
```
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'एक्काइस' instead of 'एक्काईस'.

File: lib/src/constants.dart [hunk 7]
```diff
@@ -16,46 +16,46 @@ class Constants {
...
+    "बाईस",
     "तेइस",
     "चौबीस",
-    "पचीस",
-    "छब्बीस",
-    "सत्ताइस",
```
Comment (subarnaPdl) [inline lib/src/constants.dart:36]: 'सत्ताइस' is already correct here.

File: lib/src/constants.dart [hunk 8]
```diff
@@ -16,46 +16,46 @@ class Constants {
...
-    "तीरचालीस",
-    "चौवालिस",
-    "पैंतालिस",
-    "छयालिस",
-    "सरचालीस",
-    "अरचालीस",
-    "उननचास",
+    "बयालीस",
+    "त्रिचालीस",
+    "चौवालीस",
```
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'चवालीस' instead of 'चौवालीस'.

File: lib/src/constants.dart (hunk 3 above)
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'सच्चालीस' instead of 'सड्चालीस'.

File: lib/src/constants.dart (hunk 4 above)
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'उनन्चास' instead of 'उनन्पचास'.

File: lib/src/constants.dart [hunk 9]
```diff
@@ -64,48 +64,48 @@ class Constants {
...
-    "एकसाठी",
-    "बासाठी",
-    "तीरसाठी",
-    "चौंसाठी",
-    "पैसाठी",
-    "छैसठी",
-    "सत्सठ्ठी",
-    "अर्सठ्ठी",
+    "एकसठ्ठी",
+    "बैसठ्ठी",
```
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'बयसट्ठी' instead of 'बैसठ्ठी'.

File: lib/src/constants.dart [hunk 10]
```diff
@@ -64,48 +64,48 @@ class Constants {
...
+    "त्रिसठ्ठी",
+    "चौँसठ्ठी",
+    "पैँसठ्ठी",
+    "छैसठ्ठी",
```
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'छयसट्ठी' instead of 'छैसठ्ठी'.

File: lib/src/constants.dart [hunk 11]
```diff
@@ -64,48 +64,48 @@ class Constants {
...
+    "सड्सठ्ठी",
```
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'सतसट्ठी' instead of 'सड्सठ्ठी'.

File: lib/src/constants.dart [hunk 12]
```diff
@@ -64,48 +64,48 @@ class Constants {
...
+    "अठ्सठ्ठी",
```
Comment (subarnaPdl) [inline lib/src/constants.dart]: it should be 'अठसट्ठी' instead of 'अठ्सठ्ठी'

File: lib/src/constants.dart [hunk 13]
```diff
@@ -64,48 +64,48 @@ class Constants {
...
-    "त्रीयासी",
+    "त्रियासी",
     "चौरासी",
     "पचासी",
     "छयासी",
     "सतासी",
     "अठासी",
     "उनान्नब्बे",
     "नब्बे",
-    "एकान्नब्बे",
```
Comment (subarnaPdl) [inline lib/src/constants.dart:100]: all from 'एकान्नब्बे' to 'अन्ठान्नब्बे' except 'पंचान्नब्बे' are already correct here.
'पंचान्नब्बे' can be converted to 'पन्चान्नब्बे'.

**Review by subarnaPdl — COMMENTED** [review]
I've requested some changes, please go through it once.

File: lib/src/constants.dart [hunk 14]
```diff
@@ -64,48 +64,48 @@ class Constants {
...
-    "बयान्नब्बे",
-    "त्रियान्नब्बे",
-    "चौरान्नब्बे",
-    "पंचान्नब्बे",
-    "छयान्नब्बे",
-    "सन्तान्‍नब्बे",
-    "अन्ठान्नब्बे",
```
Comment (subarnaPdl) [inline lib/src/constants.dart]: all from 'एकान्नब्बे' to 'अन्ठान्नब्बे' except 'पंचान्नब्बे' are already correct here.
'पंचान्नब्बे' can be converted to 'पन्चान्नब्बे'.

**Pull request merged**